src/components/dialogs/DialogAssociation.vue

```diff
@@ -190,6 +190,20 @@ export default {
 		}
 	},
 	methods: {
+		allowedAssociation() {
+			const mainNode = this.node
+			const targetNode = this.group.target
+
+			if (!mainNode || !targetNode) {
+				return true
+			}
+
+			if (mainNode.security !== targetNode.security) {
+				return `Association not allowed, "${targetNode._name}" security is different from "${mainNode._name}"`
+			}
+
+			return true
+		},
```

Comment:
No, use `Controller.checkAssociation` to figure out if the association is allowed and if not, why not. Then display a different message for each reason.

Reply:
The screenshot in the issue was just **one** example.

Reply:
Is that actually exported from safe somewhere?

Reply:
The idea is to make a backend api call, the backend calls driver.controller.checkAssociation and passes the result back.

This requires v13 because the method and AssociationCheckResult was only added there.